Src/C/run-time/eif_threads.h

```diff
@@ -66,10 +66,10 @@
 #define eif_thr_default_priority() 0
 #define eif_thr_min_priority() 0
 #define eif_thr_max_priority() 0
-#define eif_thr_create_with_attr(current_obj, init_func, attr)
-#define eif_thr_create_with_attr_new(current_obj, init_func, log_id, is_proc, attr)
+#define eif_thr_create_with_attr(current_obj, init_func, set_terminated_func, attr)
+#define eif_thr_create_with_attr_new(current_obj, init_func, set_terminated_func, log_id, is_proc, attr)
 /* Obsolete `eif_thr_create_with_args' but maintained to make it easy to switch between runtime versions. */
-#define eif_thr_create_with_args(current_obj, init_func, priority, policy, detach)
+#define eif_thr_create_with_args(current_obj, init_func, set_terminated_func, priority, policy, detach)
```

Comment:
No it is not except a reference in Gobo. So we could remove it in a separate commit.